lib/private/strip-stack-message.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function stripStackMessage(e) {
```

Comment:
Can we add documentation to this?  See https://github.com/node-machine/machine/blob/master/lib/private/get-combined-error-stack.js#L9-L29 for an example of what I'm getting at.